tools/pybind11NewTools.cmake

```diff
@@ -181,6 +181,13 @@ function(pybind11_add_module target_name)
   if(MSVC)
     target_link_libraries(${target_name} PRIVATE pybind11::windows_extras)
   endif()
+
+  if(ARG_OPT_SIZE)
+    target_link_libraries(${target_name} PRIVATE pybind11::opt_size)
+  endif()
+
+  # Avoid the -D${target_name}_EXPORTS definition that CMake adds by default
```

Comment:
Are these harmful, though? I would assume they do nothing if never used, and someone might expect them to be there?

Reply:
Ok, fair enough -- I'll delete them again.